### ISSUE TITLE
AWS access for Maurice Okumu

### ISSUE THREAD

**Issue by Maurice-Okumu** [department-of-veterans-affairs/va.gov-team#63830]
### COR Name

Cecila Lee

### Vendor Onboarding Representative Name

Liberty IT -  Michael G. Lovejoy

### E-QIP Transmittal Date

- [X] Check this box if you have an E-QIP transmittal date.

### Your Name

Maurice Okumu

### Your Email

maurice.okumu@thoughtworks.com

### Team, Role, and Company of the target individual

Chatbot, Software Developer , Thoughtworks

### Product Manager (PM) name and email

Alina Murphy, amurphy@thoughtworks.com

### Product Owner (PO) name and email

Luciana Morais, luciana.morais@va.gov

### Desired AWS Access

I need read/write access to dsva-vagov/va-virtual-agent space in AWS parameter store.

### Access Expiration

3/1/24

### E-QIP Transmittal Confirmation

<img width="1266" alt="eqip-transmittal" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/74667197/49b61989-b735-4baa-a499-d0b3d88258ef">

### User must exist in a roster before AWS access can be granted. See [AWS access documentation](https://vfs.atlassian.net/wiki/spaces/OT/pages/1792114735/Onboard+New+Team+Members+Granting+Access+to+Internal+Tools#AWS-Access-Request) for the steps involved.

- [x] Search for user on the [VFS Team Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)
- [ ] Or search for user on the [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)
- [ ] If user is on a VFS team but not in the VFS Team Roster, add the 'NOT YET' label and instruct them to start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] If a user is on a Platform team but not on the Platform Team Roster in Confluence, add the 'NOT YET' label and instruct them to reach out to their Product Manager to be added.
- [x] Comment in this issue saying which roster the user is listed in.

**Comment by Maurice-Okumu** [comment]
Good afternoon. I am just following up on this issue.

**Comment by ben-allen-oddball** [comment]
Found on VFS Roster

**Comment by ben-allen-oddball** [comment]
Waiting on approval from @BillChapmanUSDS

**Comment by Kshitiz-devops** [comment]
Duplicate of https://github.com/department-of-veterans-affairs/va.gov-team/issues/57347
Closing
